examples/create-example.yaml

```diff
@@ -8,8 +8,10 @@ spec:
   clusterName: "my-cluster"
   baseUrl: "https://management.azure.com/"
   authBaseUrl: "https://login.microsoftonline.com"
-  tenantId: "REPLACE_WITH_TENANT_ID"
   azureCredentialSecret: "REPLACE_WITH_K8S_SECRETS_NAME"
+  privateCluster: false,
+  linuxAdminUsername: "rancher-user",
+  linuxAdminPassword: "rancher-password",
```

Comment:
There's no `linuxAdminPassword`, it's `linuxSSHPublicKey`

Reply:
ahh yeah, you are right, but there is `sshPublicKey` option